### DISCUSSION TITLE
Python Code completion?

### DISCUSSION THREAD

**Discussion by verbesserba** [thebaselab/codeapp#1189] [Q&A]
is there code completion for python in this app?

**Comment by bummoblizard** [comment]
Yes. It works like VS Code's Python extension.

![image](https://github.com/user-attachments/assets/bd55a302-633f-46c1-abaf-e1386616be6d)

**Reply by verbesserba** [reply]
Ok thank you :D

**Comment by Jackqhr** [comment]
i'm wondering whether it is possible to add LSP support for cpp?
